pkg/resolver/multi_test.go

```
@@ -226,4 +227,15 @@ func TestResolve(t *testing.T) {
 			}
 		})
 	}
+
+	t.Run("close all", func(t *testing.T) {
+		mr.Close()
```

Error is not checked.